java/piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -354,4 +354,188 @@ public void negative() throws IOException {
       assert_().fail("Incorrect parameters passed to the checker");
     }
   }
+
+  @Test
+  public void positiveCaseWithFlagNameAsVariable() throws IOException {
+
+    ErrorProneFlags.Builder b = ErrorProneFlags.builder();
+    b.putFlag("Piranha:FlagName", "STALE_FLAG");
+    b.putFlag("Piranha:IsTreated", "true");
+    b.putFlag("Piranha:TreatmentGroup", "GROUP_A");
```

Comment:
This TreatmentGroup parameter is not needed.

Reply:
fixed